File: assignment 3/factorial.cpp
#include<iostream>
using namespace std;
class Factorial
{
    int i,fact=1,num;
    public : void display()
    {
        cout<<"enter any number:";
        cin>>num;
        for(i=1;i<=num;i++)
        {
            fact=fact*i;
        }
        cout<<fact;

    }

};
int main()
{
    Factorial f1;
    f1.display();
}

File: JAVAAssignment 3/Factorial.java
import java.util.Scanner;
class Factorial
{
public static void main(String[] args)
{
int i,fact=1,num;
System.out.println("Enter any number");
Scanner sc=new Scanner(System.in);
num=sc.nextInt();
for(i=1;i<=num;i++)
{
fact=fact*i;
}
System.out.println(fact);
}
}